### DISCUSSION TITLE
rtf writer, firstLineIndent

### DISCUSSION THREAD

**Discussion by mureva** [jgm/pandoc#9761] [Q&A]
How do I set the format of paragraphs in the main body when using the rtf writer? Specifically, paragraph spacing and first line indent? It looks from the rtf writer code like there's support for first line indent, but how do I configure it?

**Comment by jgm** [comment]
There isn't support for first line indent (I see that the code has a function that would support it, but the writer simply sets this value to 0 every time it is called!).

If we implemented #8337 you would have more control over formatting.  See https://github.com/jgm/pandoc/issues/8337#issuecomment-1264723273 for an easy project that would help me in this.
